lib/CL/devices/cuda/pocl-cuda.c

```diff
@@ -52,6 +52,7 @@ typedef struct pocl_cuda_device_data_s
   cl_ulong epoch;
   char libdevice[PATH_MAX];
   pocl_lock_t compile_lock;
+  int supports_cu_mem_host_register;
```

Comment:
Is this actually a per-host property, not per-device?

Reply:
It's a per device property AFAIK